web_widget_radio_tree/static/src/js/widget.js

```diff
@@ -0,0 +1,51 @@
+openerp.web_widget_radio_tree = function (instance) {
+
+    var QWeb = instance.web.qweb;
+
+    instance.web.list.columns.add('field.radio_tree', 'instance.web.list.RadioTreeColumn');
+
+    instance.web.list.RadioTreeColumn = instance.web.list.Column.extend({
+        _format: function (row_data, options) {
+            return QWeb.render('RadioTreeColumn', {
+                name: options.model + '_' + this.id,
+                checked: row_data[this.id].value ? {checked: ''} : {},
+            });
+        }
+    });
+
+    instance.web.form.widgets.add('radio_tree', 'instance.web.form.RadioTree');
+
+    instance.web.form.RadioTree = instance.web.form.FieldBoolean.extend({
+        template: 'RadioTree',
+        start: function() {
+            var _super = this._super.apply(this, arguments);
+            this.$checkbox = $('input', this.$el);
+            var radio_name = this.getParent().model + '_' + this.$checkbox[0].name;
+            this.$checkbox.attr('name', radio_name);
+
+            var self = this;
+            this.$el.click(_.bind(function() {
```

Comment:
use the proxy function: https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L805